wolox-books/src/app/screens/user/components/register/register.component.html

```diff
@@ -1,4 +1,4 @@
-<form [formGroup]="formGroup" (ngSubmit)="request(formGroup)" class="form">
+<form [formGroup]="formGroup" (ngSubmit)="addUser(formGroup)" class="form">
```

Comment:
```suggestion
<form [formGroup]="formGroup" (ngSubmit)="addUser()" class="form">
```
read `formGroup` inside of the method.